### DISCUSSION TITLE
ruckus/Xi3ToR500Conversion

### DISCUSSION THREAD

**Discussion by giscus** [ms264556/Hackery#10] [Comments]
# ruckus/Xi3ToR500Conversion

http://ms264556.net/ruckus/Xi3ToR500Conversion

**Comment by headcase0570** [comment]
Excellent tutorial, I've converted 4 rather useless Xclaim Xi-3 units to fully loaded Ruckus R500 mesh-enabled units. Thank you!

**Reply by headcase0570** [reply]
Update: I've been running 9 XI-3's-turned-R500's for a while now. Seeing as I've bought a load of XI-3's for less than €10,- a piece, I think it's amazing to see it work this well.

**Comment by Rink76** [comment]
Wow, thanks so much for this article! I had four working XClaims with a terrible web GUI, but after upgrading, I can manage them properly now!

**Comment by fishfish-fish** [comment]
Just a small improvement to the guide - Upload and install the R500 Solo firmware
 step.
You don't need TFTP server to upload firmware to the router. From Busybox you can do SCP to your PC (Linux in my case) and get the file. Tested recently on my XI-3:
`scp user@your.pc.ip:~/Downloads/R500_110.0.0.0.2005.bl7 /tmp/solo.img`

**Comment by vitalius** [comment]
Thanks for the article, my Xi-3 is alive again!

**Comment by fishfish-fish** [comment]
And now as they transition to new CAM auth system on the https://unleashed.ruckusnetworks.com/ - it require 200.15+ firmware version...

**Comment by headcase0570** [comment]
A question: The XI3-turned R500 seems to poll captive.apple.com at a regular interval, if it's not detected, due to a WAN disconnect, the units running the R500 firmware will drop all connected clients until WAN connection is confirmed through, again, polling captive.apple.com. This is a huge pain in the @$$. I live in an area with an unreliable WAN connection, and rebuilding the Unleashed mesh can take anywhere from 5 minutes to over an hour. Every single time my WAN drops for a few seconds. So far I've been unable to find a method of disabling the polling, or have it poll a local device, like the internal address of the router. Have you got any insight into settings that I could change to keep WLAN clients connected to the converted AP's when the WAN goes down/captive.apple.com becomes unreachable?

**Reply by ms264556** [reply]
Yeah, they added an option to disable this check in a later Unleashed release, which you can't run on an R500.

You can pick one of the sites it polls, and add a host override to your gateway router's DNS service, pointing to 127.0.0.1 (assuming your ISP router is providing a DNS forwarder and allows host overrides).
I think Unleashed also polls `baidu.com`. Since this site is next to useless outside China, that's the one I'd override.

The host override functionality is e.g. in *Network → Hostnames* on OpenWRT, *DNS Resolver → Host Overrides* on pfSense.

**Reply by remixedcat** [reply]
is that only if you have the captive portal enabled? or does that also happen to a regular SSID w no captive portal?

**Reply by ms264556** [reply]
Always. It's unrelated to captive portal.

**Reply by remixedcat** [reply]
ugh that sucks... I use a unifi cloud gateway router and unifi switch ultra 60w switch how do I do the override?

**Comment by stefantheiner-dev** [comment]
This was a lot of fun. Thank you fot the perfect article :)

**Comment by remixedcat** [comment]
Successful on my xi3 in the big 2026! had to use a different tftp server but got it. Unleashed firmware as well!
